src/components/course/EnrollButton.jsx

```diff
@@ -56,7 +57,8 @@ export default function EnrollButton() {
           course_id: key,
           enterprise_customer_uuid: enterpriseConfig.uuid,
           next: `${process.env.LMS_BASE_URL}/courses/${key}/course`,
-          failure_url: global.location,
+          // Redirect back to the LP dashboard with a failure query param
+          failure_url: `${global.location.host}/${enterpriseConfig.slug}/?${ENROLLMENT_FAILED}=true`,
```

Comment:
from a UX perspective, it feels a bit odd to me to redirect to the dashboard page on DSC failure, when the user was previously on the course page. this means if the user wants to re-enroll and accept DSC instead of denying it, they now have to find the course again via the search page before trying to enroll again (2-step process).

A couple thoughts:
* Is there a way to render the error notification on the Course page that looks decent?
* Given I don't know what a good design is right now for displaying this error notification directly on the Course page, another alternative might be to potentially link to the course directly from the notification? That way, the user doesn't need to re-find the course.

Reply:
I agree that it feels weird from a UX perspective, but @brittneyexline said that if we wanted to change the UX from the decided homepage we could do that later. It looks fine (to me) to render a status alert on the Course page, but sounds like we might hold off on that

Reply:
i remember that being our decision, but feel free to ask Joe and B Cool.

Reply:
I think it's fine as is for initial release, only to cut scope, but am concerned the UX is not great due to adding unnecessary, extra steps for the user to accomplish their goal of enrolling in a course; I will follow-up with Joe and file a ticket in the product discovery backlog.

Reply:
Talked with Joe and he want to redirect back to the course page. I will update this PR to do that

Reply:
> Talked with Joe and he wants to redirect back to the course page. I will update this PR to do that

👍 Thanks for reaching out to Joe about this.